rust/src/client.rs

```diff
@@ -267,8 +267,8 @@ pub(crate) fn running_in_container() -> bool {
     ostree_ext::container_utils::running_in_container()
 }
 
-pub(crate) fn ostree_path_exists() -> bool {
-    Path::new("/ostree/repo/").exists()
+pub(crate) fn is_bare_split_xattrs() -> CxxResult<bool> {
```

Comment:
This could return a plain `-> bool`, maybe?
Looking at the consumers, I think we don't to forward up errors from here.

Reply:
I see an example like https://github.com/coreos/rpm-ostree/pull/3285#discussion_r772676280, https://github.com/coreos/rpm-ostree/blob/main/rust/src/lib.rs#L444 & https://github.com/coreos/rpm-ostree/blob/main/rust/src/live.rs#L556 following the same kind of workflow. Is there a difference I am missing?

Reply:
Ah I'm sorry, I didn't see that Colin pushed it in this direction. Forget what I said then, I'll let him steer this through review.